mkdocs.yml

```
@@ -224,7 +224,11 @@ nav:
           - Adobe:
             - Adobe Analytics: data/sources/adobe-analytics.md
             - Adobe Tag Manager: data/sources/adobe.md
-          - Google Tag Manager: data/sources/google-tag-manager.md
+          - Google Tag Manager: 
+            - Quickstart Guide: data/sources/google-tag-manager.md
+            - Amplitude Analytics Browser SDK: data/sources/google-tag-manager-client.md
```

```suggestion
            - Analytics Browser SDK: data/sources/google-tag-manager-client.md
```
Setting an alternate title here to keep it from breaking across lines in left nav.

---

Changed them to `Client-side`, `Server-side`, `Client-side (Legacy)` respectively since the name is long and not easy to remember.